compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.h

```diff
@@ -18,6 +18,7 @@ namespace mlir::iree_compiler::AMDAIE {
 enum class AIEPassPipeline : int32_t {
   PackPeelPipeline = 0,
   PadPackPipeline = 1,
+  ConvDecomposePipeline = 2,
   None = 2
```

Comment:
None = 2 (is that what I'm seeing)? 

I think it's not necessary to give all the AIEPassPipeline names explicit numbers, having just PackPeelPipeline = 0 is enough, the others will be contiguous from there.

Reply:
Looks like we are not even using the values. I have deleted the assigned numbers.